lib/sass/script/value/callable.rb

```diff
@@ -0,0 +1,48 @@
+module Sass::Script::Value
+  # A SassScript object representing a null value.
+  class Callable < Base
+    # Constructs a Callable value for use in SassScript.
+    #
+    # @param callable [Sass::Callable] The callable to be used when the
+    # callable is called.
+    def initialize(callable)
+      super(callable.dup)
+    end
+
+    def to_s(opts = {})
+      raise Sass::SyntaxError.new("#{to_sass} isn't a valid CSS value.")
+    end
+
+    def inspect
+      to_sass
+    end
+
+    def to_sass
+      "#{value.type}-reference(#{value.name})"
+    end
+
+    def plus(other)
+      raise Sass::SyntaxError.new("Addition is not defined for values of type: Callable")
+    end
+
+    def minus(other)
+      raise Sass::SyntaxError.new("Subtraction is not defined for values of type: Callable")
+    end
+
+    def div(other)
+      raise Sass::SyntaxError.new("Division is not defined for values of type: Callable")
+    end
+
+    def unary_plus
+      raise Sass::SyntaxError.new("Positive is not defined for values of type: Callable")
+    end
+
+    def unary_minus
+      raise Sass::SyntaxError.new("Negation is not defined for values of type: Callable")
+    end
+
+    def unary_div
+      raise Sass::SyntaxError.new("Division is not defined for values of type: Callable")
+    end
```

Comment:
The superclass impl gave a really confusing error:

```
SyntaxError: function-reference(asdf) isn't a valid CSS value.
```

As a user, whether something is a css value doesn't explain to me why I can't do math on it. I can't "do math" on any css value. I guess the proper thing is to file a new issue and address the bad error message properly for all types that don't "math".